library/list.h

```diff
@@ -17,7 +17,7 @@ static inline void prefetch(const void *x __attribute__ ((unused))) {;}
  *
  */
 #define container_of(ptr, type, member) ({			\
-        const typeof( ((type *)0)->member ) *__mptr = (ptr);	\
+        const __typeof__( ((type *)0)->member ) *__mptr = (ptr);	\
```

Comment:
why did you changed `typeof` to "__typeof__"?

Reply:
Good question. Actually it's from already merged commit. I don't know how it got here. The same is for USE_UNLOCKED_IO
https://github.com/interiorem/eblob/commit/b7d4fec97fadcb5079642cb1c914b21a14b056c1

Reply:
fixed that